gpiozero/boards.py

```diff
@@ -2063,15 +2065,13 @@ def __init__(self, socket=None, initial_value=False, pin_factory=None):
             raise EnergenieSocketMissing('socket number must be provided')
         if not (1 <= socket <= 4):
             raise EnergenieBadSocket('socket number must be between 1 and 4')
-        if initial_value is None:
-            raise EnergenieBadInitialValue("initial value can't be None")
         self._value = None
         super(Energenie, self).__init__(pin_factory=pin_factory)
         self._socket = socket
         self._master = _EnergenieMaster(pin_factory=pin_factory)
         if initial_value:
             self.on()
-        else:
+        elif initial_value is not None:
```

Comment:
Sorry to be such a nitpicker, but although what you're proposing here is "more compact", I still prefer the "more explicit" version that I suggested:
```python3
if initial_value is not None:
    if initial_value:
        self.on()
    else:
        self.off()
```

I guess it's up to @bennuttall really though!